README.md

```diff
@@ -39,16 +56,14 @@ base to provide opportunity to use it in information processing and knowledge cr
 the help of LaTex tools in SCn-code representation. To see more information about the OSTIS Technology web-oriented 
 platform implementation:
 
-## Docker
-We're providing a Docker image to build the documentation. Head to [Installing with Docker](https://docs.docker.com/get-started/) 
-section of our docs to try it out!
+We're providing a Docker image to build the documentation.
 
 To do that, run docker image:
 ```sh
-cd ostis-web-platform/docs/scn/
-docker run -v </full/path/to/ostis-web-platform>:/workdir ostis/scntex-builder 'cd docs && pdflatex -interaction=nonstopmode main.tex'
+docker run -v <path/to/ostis-web-platform/>:/workdir ostis/scntex-builder 'cd docs && pdflatex -interaction=nonstopmode main.tex'
```

Comment:
Is it right? May be it won't work, because this script must be started from ostis-web-platform/docs/scn/

Reply:
100% works, tested it myself. It doesn't matter as long as you're using the absolute path to `ostis-web-platform` to mount it